cmd/kar-controllers/app/options/options.go

```diff
@@ -30,6 +30,10 @@ type ServerOption struct {
 	Dispatcher	bool
 	AgentConfigs 	string
 	SecurePort	int
+	DynamicPriority	bool  // If DynamicPriority=true then no preemption is allowed by program logic
+	Preemption 		bool  // Preemption is not allowed under DynamicPriority
+	SendUpdate		bool  // Send QueueJobStatus Updatae() to Etcd.  Not essential for Controller functions
```

Comment:
Is `SendUpdate` for performance evaluation?  Is this field for dynamic priority policy only?  Just trying to get a sense of how this field is used to determine if we should just remove any non-essential updates overall.

Reply:
When there is a change to AppWrapper.Status, SendUpdate=true triggers cc.arbclients.ArbV1().AppWrappers(qj.Namespace).Update(qj) to update Etcd.  This allows users to see the changes using kubectl command.  For DynamicPriority policy, SystemPriority changes over time.  A job's QueueJobState (Queueing, HeadOfLine, Rejoining, ...) also changes over time.  SendUpdate option allows controller to provide these information to the users.  If SendUpdate=false then no one can tell what happens to a job until it is deployed.

Reply:
I would expect that changes to the AppWrapper state should always be reflected in etcd.  Is there cases when the states should not be reflective in etcd?  Is it that the updates we're trying to limit here is for the dynamic priority changes because of the frequency and the performance limits of the apiserver?

Reply:
Per discussion, SendUpdate will always be true.  Users will not be given the option of not sending updates to Etcd, despite possible performance implications.

Reply:
Agreed, the key here is to have the right balance between usability and performance.  Sending updates to the API Server does add additional overhead to the controller so lets continue to find ways to reduce overhead while still maintaining a good balance of usability.  Thanks!